### PR TITLE
8923 tele op drive

### PR DESCRIPTION
**Pull request by myriantics**
We created the TeleOp split arcade drive code for 8923's 2021-2022 robot.

### CODE REVIEW

File: Team8923_2021/src/main/java/org/firstinspires/team8923_2021/MasterTeleOp.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package org.firstinspires.team8923_2021;
+
+
```
Comment (malavika1) [inline Team8923_2021/src/main/java/org/firstinspires/team8923_2021/MasterTeleOp.java]: You shouldn't have multiple blank lines in a row

File: Team8923_2021/src/main/java/org/firstinspires/team8923_2021/TeleOpCompetition.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.firstinspires.team8923_2021;
+
+
```
Comment (malavika1) [inline Team8923_2021/src/main/java/org/firstinspires/team8923_2021/TeleOpCompetition.java]: Also here you shouldn't have multiple blank lines in a row

File: Team8923_2021/src/main/java/org/firstinspires/team8923_2021/Toggle.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    }
+
+    public boolean toggle(boolean button)
+    {
+        if(press)
+        {
+            toggleState = !toggleState;
+        }
+        if(! prevState && button)
```
Comment (malavika1) [inline Team8923_2021/src/main/java/org/firstinspires/team8923_2021/Toggle.java:19]: Typically a new if statement block should have a new line between the old block

**Review by malavika1 — APPROVED** [review]

**Review by sanjsaini — APPROVED** [review]
This looks great!

File: Team8923_2021/src/main/java/org/firstinspires/team8923_2021/MasterTeleOp.java (hunk 1 above)
Comment (myriantics) [reply Team8923_2021/src/main/java/org/firstinspires/team8923_2021/MasterTeleOp.java]: Resolved

**Pull request merged**